src/components/button/Button.jsx

```diff
@@ -0,0 +1,18 @@
+import './Button.css'
+import {ArrowRight} from "@phosphor-icons/react";
+
+function Button({label, name, onClick, arrow}) {
```

Comment:
Op basis van `true` of `false` die binnenkomt onder `arrow` ga je bepalen of je het icoon wel of niet wil weergeven. Bij _boolean_ properties leest het altijd het fijnst als je de property `has...` of `is...` noemt. Denk bijvoorbeeld aan `hasIcon`, dat maakt het een stuk duidelijker voor collega's!

Reply:
Aangepast